cli/internal/cmd/telemetry/utils.go

```diff
@@ -0,0 +1,28 @@
+package telemetry
+
+import "github.com/rudderlabs/rudder-iac/cli/internal/telemetry"
+
+const (
+	CommandExecutedEvent = "Command Executed"
```

Comment:
Hey @fxenik , do you think this event definition looks good ? Or should I change it to `CLI Command Executed` ?

Reply:
Command Executed sounds fine, but if you think we need to add CLI in front from extra clarity let's do it

Reply:
I think CLI Command Executed might have better context in the downstream destination.